apis/tempo/v1alpha1/auth.go

```diff
@@ -22,7 +22,7 @@ type TenantsSpec struct {
 	// +required
 	// +kubebuilder:validation:Required
 	// +kubebuilder:default:=static
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:select:static","urn:alm:descriptor:com.tectonic.ui:select:dynamic"},displayName="Mode"
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:select:static","urn:alm:descriptor:com.tectonic.ui:select:openshift"},displayName="Mode"
```

Comment:
do we actually need this, shouldn't openshift pick up the enum definition here:
```
// +kubebuilder:validation:Enum=static;openshift
type ModeType string
```
?

Reply:
I would say yes, to compare it on other places in the code.

Reply:
I just removed the `+operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors....` line and can't see any difference in OpenShift:
![image](https://github.com/grafana/tempo-operator/assets/538011/a3acec52-67fe-4858-8a8d-34f5594f5815)